pandas/tests/groupby/test_groupby.py

```diff
@@ -2732,6 +2732,16 @@ def h(df, arg3):
 
         assert_series_equal(result, expected)
 
+    def test_empty_dataframe_groupby(self):
+        # GH8093
+        df = pd.DataFrame(columns=['A', 'B', 'C'])
+
+        result = df.groupby('A').sum()
+        expected = pd.DataFrame(columns=['B', 'C'], dtype=np.float64)
```

Comment:
`DataFrame` is already imported, so can remove the `pd.` to make things a little cleaner